tools/cellprofiler/starting_modules.xml

```diff
@@ -230,14 +230,20 @@ def write_groups():
 
     _str = "\nGroups:[module_num:4|svn_version:\\'Unknown\\'|variable_revision_number:2|show_window:False|notes:\\x5B\\\'The Groups module optionally allows you to split your list of images into image subsets (groups) which will be processed independently of each other. Examples of groupings include screening batches, microtiter plates, time-lapse movies, etc.\\'\\x5D|batch_state:array(\\x5B\\x5D, dtype=uint8)|enabled:True|wants_pause:False]\n"
 
-    _str += FOURSPACES + "Do you want to group your images?:%s\n" % groups["con_groups"]["group_images"]
+    group_images =  groups["con_groups"]["group_images"]
+
+    _str += FOURSPACES + "Do you want to group your images?:%s\n" % group_images
     _str += FOURSPACES + "grouping metadata count:1\n"
-    _str += FOURSPACES + "Metadata category:%s\n" % groups["con_groups"]["group_category"]
+
+    if group_images == "Yes":

```

Comment:
Sorry, I tested in EU, not this branch... it's fixed here, left? :)

Reply:
ye..It's fixed here. 
I will test the multitiff in our environment tomorrow..

Reply:
cool thanks!!